tests/test_yaml/test9.expected.valohai.yaml

```diff
@@ -0,0 +1,7 @@
+- step:
+    name: train
+    image: tensorflow/tensorflow:2.4.0-gpu
+    command:
+    - pip install -r requirements.txt
+    - python {source-path} {parameters}
+    source-path: ./test9.py
```

Comment:
okok so it comes from the filename `tests/test_yaml/test9.py`

took me a while to figure out WTF is happening :laughing:

Reply:
yeah the testing harness is somewhat, hm, indirect